pkg/gofr/datasource/opentsdb/interface.go

```diff
@@ -0,0 +1,53 @@
+package opentsdb
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"time"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+type Conn interface {
```

Comment:
Also, do you need this interface to be exported?

I didn't find where it's used

Reply:
it is being used to write mocks for test...I will make it unexported tho.